pfcpiface/web_service.go

```diff
@@ -50,7 +50,7 @@ func (c *ConfigHandler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 	case "PUT":
 		fallthrough
 	case "POST":
-		body, err := ioutil.ReadAll(r.Body)
+		body, err := io.ReadAll(r.Body)
```

Comment:
Would like to understand reason behind this changes

Reply:
The ioutil package was deprecated. Here some text from the ioutil package (https://pkg.go.dev/io/ioutil):
```
Package ioutil implements some I/O utility functions.

Deprecated: As of Go 1.16, the same functionality is now provided by package io or package os, and those implementations should be preferred in new code. See the specific function documentation for details.
```